tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3704,6 +3720,13 @@ def runTest(self):
                 for cntr in egress_counters:
                     assert (xmit_counters[cntr] > xmit_counters_base[cntr])
 
+            # voq ingress drop
+            if platform_asic and platform_asic == "broadcom-dnx":
+                voq_index = pg - 2
```

Comment:
Please share more info on this logic

Reply:
As we discussed, the pg value is set by adding 2 (pg = int(self.test_params['pg']) + 2) in the  initialization. Hence  reducing  2 here to get the correct voq_index